ts/features/fims/common/analytics/__tests__/index.test.ts

```diff
@@ -26,10 +26,9 @@ const organizationNames = [undefined, "organization name"];
 const referenceReason = "The reason";
 const referenceServiceId = "01J9RSWBB4VSHVRJSY33XGA6YH" as ServiceId;
 const serviceNames = [undefined, "service name"];
-const sources: ReadonlyArray<"message_detail" | "service_detail"> = [
-  "message_detail",
-  "service_detail"
-];
+const sources: ReadonlyArray<
+  "message_detail" | "service_detail" | "credential_details"
+> = ["message_detail", "service_detail", "credential_details"];
```

Comment:
```suggestion
const sources: = ["message_detail", "service_detail", "credential_details"] as const;
```
This is a cleaner way to have a strong type for the `sources` array. What do you think?

Reply:
I agree with you, thanks for the suggestion